performance/patient-encounter-observation-bundle-create.js

```diff
@@ -0,0 +1,102 @@
+/* global open __ENV */
+/* eslint no-undef: "error" */
+
+import http from 'k6/http'
+import { check } from 'k6'
+
+const patientResource = JSON.parse(open('./resources/patient.json'))
+const encounterResource = JSON.parse(open('./resources/encounter.json'))
+const heightObservation = JSON.parse(open('./resources/heightResource.json'))
+const weightObservation = JSON.parse(open('./resources/weightResource.json'))
+
+const BASE_URL = __ENV.BASE_URL || 'http://localhost:3447'
+
+export const options = {
+  vus: 10,
+  iterations: 1000,
+  thresholds: {
+    http_req_receiving: ['p(95)<100'],
+    http_req_duration: ['p(95)<100']
+  },
+  noVUConnectionReuse: true,
+  discardResponseBodies: true
+}
+
+let patientResourceFullUrl
+let encounterResourceFullUrl
+
+const bundleResource = {
+  resourceType: 'Bundle',
+  type: 'transaction',
+  timestamp: Date.now(),
+  entry: [
+    {
+      fullUrl: patientResourceFullUrl,
+      resource: patientResource,
+      request: {
+        method: 'POST',
+        url: `Patient`
+      }
+    },
+    {
+      fullUrl: encounterResourceFullUrl,
+      resource: encounterResource,
+      request: {
+        method: 'POST',
+        url: `Encounter`
+      }
+    },
+    {
+      resource: heightObservation,
+      request: {
+        method: 'POST',
+        url: `Observation`
+      }
+    },
+    {
+      resource: weightObservation,
+      request: {
+        method: 'POST',
+        url: `Observation`
+      }
+    }
```

Comment:
Lets add one more entry here to make a GET request for the Observations that were created, and verify the response bundle has two Observation entries being returned.